src/detect.h

```diff
@@ -909,6 +909,8 @@ typedef struct DetectEngineThreadCtx_ {
     uint8_t *base64_decoded;
     int base64_decoded_len;
     int base64_decoded_len_max;
+    uint8_t alert_flags;
```

Comment:
This would also make the target keyword much simpler as it wouldn't need it's own SigMatch and all.